stl/CMakeLists.txt

```diff
@@ -397,13 +398,18 @@ set(STATIC_SOURCES
     ${SOURCES_SATELLITE_CODECVT_IDS}
 )
 
+add_library(std_init_once_begin_initialize OBJECT IMPORTED)
+add_library(std_init_once_complete OBJECT IMPORTED)

```

Comment:
@cbezault Why two object libraries instead of one?

Reply:
No reason. We could make them a single library and give it a nice logical name.